src/panels/config/tags/dialog-tag-detail.ts

```diff
@@ -191,13 +195,43 @@ class DialogTagDetail extends LitElement implements HassDialog {
     }
   }
 
+  private async _generateQR() {
+    let canvas = await QRCode.toCanvas(
+      `https://home-assistant.io/tag/${this._params?.entry?.id}`,
+      {
+        width: 180,
+        errorCorrectionLevel: "Q",
+      }
+    );
+    let context = canvas.getContext("2d");
+
+    let imageObj = new Image();
+    imageObj.src = "/static/icons/favicon-192x192.png";
```

Comment:
We should probably make this a const somewhere. Atleast top of the file? could definitely change

Reply:
There could probably be a `src/resources/static.ts`or something to keep all of this kind of stuff, but there really aren't that many as it turns out:
```
~/c/h/frontend > git grep "/static/" src/
src/common/dom/setup-leaflet-map.ts:  Leaflet.Icon.Default.imagePath = "/static/images/leaflet/images/";
src/common/dom/setup-leaflet-map.ts:  style.setAttribute("href", "/static/images/leaflet/leaflet.css");
src/components/ha-icon.ts:    const iconPromise = fetch(`/static/mdi/${chunk}.json`).then((response) =>

src/html/_header.html.template:<link rel='icon' href='/static/icons/favicon.ico'>
# ... html templates (removed a couple of lines)

src/layouts/ha-init-page.ts:        <img src="/static/icons/favicon-192x192.png" height="192" />
src/panels/config/info/ha-config-info.ts:              src="/static/icons/favicon-192x192.png"
src/panels/config/tags/dialog-tag-detail.ts:const QR_LOGO_URL = "/static/icons/favicon-192x192.png";
src/panels/lovelace/cards/hui-media-control-card.ts:        background-image: url("../static/images/card_media_player_bg.png");
src/panels/lovelace/components/hui-image.ts:        background: grey url("/static/images/image-broken.svg") center/36px

src/resources/roboto.js:  url(/static/fonts/roboto/Roboto-Thin.woff2) format("woff2");
# ... font stuff (removed a couple of lines)...

src/util/hass-translation.ts:  const response = await fetch(`/static/translations/${fingerprint}`, {
```

The media-control-card css is an odd one out, though. Expect a PR.